config_table.tf

```diff
@@ -118,12 +118,12 @@ resource "aws_kms_ciphertext" "redshift_password" {
 }
 
 resource "aws_kms_alias" "lambda_alias" {
-  name = "alias/LambaRedshiftLoaderKey"
+  name = "alias/ControlshiftLambaRedshiftLoaderKey"
```

Comment:
Have you tested that this works? I think this is defined in the upstream code re: how it reads the KMS encryption info.

Reply:
Ah, yes, you are correct.